css/index.css

```diff
@@ -1,3 +1,289 @@
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+.container{
+  margin: 0 auto;
+  width: 876px;
+}
+.navigation {
+	display: inline-block;
+  position:fixed;
+  background-color: white;
+}
+.navigation a{
+  display: inline-block;
+  width: 100px;
+  height: 20px;
+  text-decoration: none;
+  padding: 10px;
+  color: black;
+
+}
+.logo {
+	display: inline-block;
+	vertical-align: top;
+	padding: 7px;
+	margin-left: 49px;
+}
+.header h1{
+	display: inline-block;
+    font-size: 50px;
+    text-align: center;
+    width: 49.5%;
+    padding: 116px;
+    vertical-align: top;
+}
+header{
+  margin:20px;
+}
+.headerImage {
+	display: inline-block;
+	width: 49.5%;
+	padding: 50px;
+}
+button{
+	display: inline-block;
+	text-decoration: none;
+	font-size: 20px;
+	border: 1px solid black;
+	padding: 10px;
+}
+
+.divider{
+  border-top: 2px solid black;
+  border-bottom: 2px solid black;
+}
+.top1{
+	width: 49.5%;
+	display: inline-block;
+	padding: 30px;
+	vertical-align: top;
+}
+.top2{
+	width: 49.5%;
+	display: inline-block;
+	padding: 33px;
+	vertical-align: top;
+}
+.bottom1{
+	display:inline-block;
+	width:33%;
+	padding: 30px;
+	vertical-align: top;
+}
+.bottom2{
+	display:inline-block;
+	width:33%;
+	padding: 30px;
+	vertical-align: top;
+}
+.bottom3{
+	
+	display:inline-block;
+	width:30%;
+	padding: 30px;
+	vertical-align: top;
+}
+.contact {
+	padding-top:50px;
+}
+.contact p{
+	padding:10px 0px 10px 0px;
+}
+.copyright{
+	display:block;
+	text-align: center;
+	padding: 50px 0px 50px 0px;
+}
+
+@media(max-width: 600px){
+    .container{
+        width: 100%;
+    }         
+    
+    nav{
+      display: flex;
+      flex-direction: column-reverse;
+      justify-content: space-between;
+      width: 100%;
+      padding-bottom: 5%;
+      color: gray;
+    }
+
+    .logo{
+        width: 40%;
+        align-self: center;
+    }
+    li{
+        padding: 1%;
+    }
+
+    ul{
+      display: flex;
+      justify-content: space-evenly;
+    }
+    
+    .headerImage img{
+        display: none;
+    }
+
+    .middle-img{
+        width: 100%;
+    }
+
+    .banner{
+
+        flex-direction: column;
+    }
+    
+    .header{
+        display: flex;
+        flex-direction:column;
+        width: 100%;
+        justify-content: space-evenly;
+    }            
+
+    section {
+        display: flex;
+        align-content: space-between;
+    }
+}
+
+    @media(max-width: 500px){
+        .container{
+            width: 100%;
+        }
+
+        nav{
+            width: 100%;
+            display: flex;
+            flex-direction: column-reverse;
```

Comment:
What `<nav>` element are you setting here? I can't seem to find one in your HTML file.

Perhaps you meant your `div` with `class="navigation"`?